### DISCUSSION TITLE
install both, Flatpak und "Debian Backport" version in parallel

### DISCUSSION THREAD

**Discussion by Golddouble** [borgbase/vorta#963] [General]
Until now I have always used the flatpack version of Vorta.
Today I have by accident found Vorta as "debian backport" in my Linux distribution.

So I would like to install this "debian backport" version, but keep the flatpack version until I am sure, that the "Debian backport" version works without issues.

Question:
Can I install Vorta "debian packport" version in parallel to the flatpack version? Or will the install of the "Debian backport" version destroy the flatpack version?

Thank you.

**Comment by Hofer-Julian** [comment]
You can install both.
They will also store its data in separate directories.

I would not recommend though to *run* them at the same time

**Comment by Golddouble** [comment]
OK. I think the crucial point is the Borg version. In my flatpack I have Borg 1.1.15. But when I install the backport Vorta, I have Borg from my system and that is only 1.1.9 .

So I guess I should wait with changing the repository to Backport until I have Borg 1.1.15 in my system. (?)

Another thing is: I have only one Vorta entry in my menu. Although I should now have two entries. Vorta from flatpack and Vorta from Backport. When I then start Vorta from the menu Flatpack Vorta starts.

**Reply by sten0** [reply]
Hi @Golddouble, Borg 1.1.15 is also available from backports; to use it, do `sudo apt install -t buster-backports borgbackup`.  1.1.16 is in testing, so is a candidate for backporting; feel free to request an update on the backports mailing list and CC locutusofborg (his email is available in the changelog `apt changelog borgbackup`).  No further action will be required when upgrading to bullseye (Debian 11)--no awkward PPA dance.  This assumes you're running Debian, or a 100% compatible Debian-derived distribution, eg: not Ubuntu or its derivatives which may require a source package rebuild of borgbackup, vorta, and possibly rebuilds of newer dependencies.

**Comment by Golddouble** [comment]
I have found it now, thank you for sharing @sten0.

Is it new in backports? I haven't seen it before.

**Comment by sten0** [comment]
Golddouble ***@***.***> writes:

> I have found it now, thank you for sharing @sten0.
>

You're welcome!  Enjoy :-)

> Is it new in backports? I haven't seen it before.
>

Here is a resource to find the answer for this:

https://tracker.debian.org/pkg/borgbackup

Cheers!
